src/main/java/com/xiaomi/infra/pegasus/rpc/async/MetaSession.java

```diff
@@ -18,11 +18,7 @@
 
 public class MetaSession extends HostNameResolver {
   public MetaSession(
-      ClusterManager manager,
-      String[] addrList,
-      int eachQueryTimeoutInMills,
-      int defaultMaxQueryCount,
-      EventLoopGroup g)
+      ClusterManager manager, String[] addrList, int defaultMaxQueryCount, EventLoopGroup g)
```

Comment:
Why you remove `eachQueryTimeoutInMills` and add parameter called timeout for `asyncQuery`?

Reply:
In this pr, we should add some interface in MetaSession, which name are query and asyncQuery, which should have a timeout parameter.
And `eachQueryTimeoutInMills` is used in the old version of query and asyncQuery, which have no timeout parameter. So If we retain `eachQueryTimeoutInMills`, we should retain these old interfaces.
In my completetion, I delete these old interfaces.  We should pass timeout to default value or specific value, when you call these interfaces. In this way, there is no need to retain these two versions of interfaces.